corehq/apps/analytics/static/analytix/js/noopMetrics.js

```diff
@@ -0,0 +1,40 @@
+define('analytix/js/noopMetrics', [], function () {
+    /**
+     * Records an event and its properties
+     * @param {string} name - Name of event to be tracked
+     * @param {object} properties - (optional) Properties related to the event being tracked
+     * @param {function} callbackFn - (optional) Function to be called after the event is tracked.
+     * @param {integer} timeout - (optional) Timeout for safe callback
+     */
+    var trackEvent = function (name, properties, callbackFn, timeout) {
```

Comment:
Curious why these arguments are needed? Passing extra arguments to a function is not an error in JS, and given that the args are documented in the docstring, it seems reasonable to omit them here.

Reply:
Although, I do wonder if it'll just say something like "Parameter properties described in JSDoc does not appear in function signature" if I do that. I guess we'll see!